src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/HttpJupyterDAO.scala

```diff
@@ -28,8 +30,8 @@ class HttpJupyterDAO(val clusterDnsCache: ActorRef)(implicit system: ActorSystem
     }
   }
 
-  protected def getTargetHost(googleProject: GoogleProject, clusterName: ClusterName): Future[GetClusterResponse] = {
+  protected def getTargetHost(googleProject: GoogleProject, clusterName: ClusterName): Future[HostStatus] = {
     implicit val timeout: Timeout = Timeout(5 seconds)
-    (clusterDnsCache ? GetByProjectAndName(googleProject, clusterName)).mapTo[GetClusterResponse]
+    clusterDnsCache.projectNameToHost(DnsCacheKey(googleProject, clusterName)).mapTo[HostStatus]
```

Comment:
Getting the compilation warning
```
[warn] /Users/kyuksel/github/leonardo/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/HttpJupyterDAO.scala:35:21: method apply in trait LoadingCache is deprecated: see corresponding Javadoc for more information.
[warn]     clusterDnsCache.projectNameToHost(DnsCacheKey(googleProject, clusterName)).mapTo[HostStatus]
[warn]                     ^
```
which can be fixed by using `get` instead of `apply`:
```scala
clusterDnsCache.projectNameToHost.get(DnsCacheKey(googleProject, clusterName)).mapTo[HostStatus]
```